frontend/src/components/Layout/Header/styled.ts

```diff
@@ -158,7 +151,7 @@ export const Search = styled(Input)<CoursesPageProps>(({ pagename, menuopen }) =
     fontSize: '24px',
     margin: '15px 20px 15px 0px',
   },
-  '@media(min-width: 1570px)': {
+  '@media(min-width: 1650px)': {
     width: '730px!important',
```

Comment:
There are some typos like `width: '730px!important',`. Is it still working with this directive?

Reply:
Yes, it's working https://prnt.sc/26tt1t1